lib/looker.ts

```diff
@@ -0,0 +1,70 @@
+import { LookerNodeSDK } from "@looker/sdk-node"
+import { IDashboardElement, IWriteQuery } from "@looker/sdk"
+
+/**
+ *
+ * @type {string} Local configuration file name, one directory above
+ */
+
+console.log('LOOKER ENABLED: ', process.env.IS_LOOKER_ENABLED);
+
+export async function getAWDashboardElement0(SDK: any): Promise<IDashboardElement> {
+  const dashboardId = "1471";
+//  await(getDBFilters())
+
+  const elements: IDashboardElement[] = await SDK.ok(SDK.dashboard_dashboard_elements(dashboardId));
+  return elements[0];
+}
+
+export async function runEventCountQuery(filters: any): Promise<any>{
+  const SDK = LookerNodeSDK.init40()
+  const me = await SDK.ok(SDK.me());
+
+  const element0 = await getAWDashboardElement0(SDK)
+  const origQuery = element0.query as IWriteQuery
+
+  // take the query from the original dashboard
+  const newQueryBody = structuredClone(origQuery)
+  delete newQueryBody.client_id // must be unique per-query
+
+  // override the filters
+  newQueryBody.filters = Object.assign(
+    {
+      'event_counts.message_id': '%FAKESPOT_OPTIN_DEFAULT%',
+      'event_counts.submission_timestamp_date': '2 days'
+    },
+    filters
+  )
+
+  // console.log("filters: ", filters)
+  // console.log("newQueryBody.filters: ", newQueryBody.filters)
```

Comment:
Should I leave these console.log statements here for debugging or can I remove these?

Reply:
Feel free to remove them.  At some point we're going to want a real, more permanent logging strategy, but for now we might as well keep the code clean.